articles/relnotes/index.md

```diff
@@ -25,7 +25,20 @@ dotnet tool update -g Microsoft.Quantum.IQSharp
 > * In order to work with a solution in Visual Studio, all projects contained in the solution need to be in the same folder as the solution or in one of its subfolders.  
 > * References between projects migrated to 0.6 and higher and projects using older package versions are **not** supported.
 
-# Version 0.9 (*PackageReference 0.9.1908.2906*)
+# Version 0.10 (*PackageReference 0.10.XXX.YYY*)
+
+*Release date: September 26th, 2019*
+
+This release contains the following:
+
+- New support for Q# code completion in Visual Studio 2019 (versions 16.3 & later) & Visual Studio Code
+- New support for installing the QDK for Python using Conda packages, new installation instructions can be found [here](xref:microsoft.quantum.install) 
+- New [Quantum Kata](https://github.com/Microsoft/QuantumKatas) for quantum adders
```

Comment:
I would not include new kata in the release notes - technically the new katas were added while still on 0.9. In the blog post it makes sense to announce them, but that's more of a "new and interesting" section than actual part of the release.